curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/643c14c8027369027334e802.md

```diff
@@ -0,0 +1,156 @@
+---
+id: 643c14c8027369027334e802
+title: Step 4
+challengeType: 0
+dashedName: step-4
+---
+
+# --description--
+
+You want to get the response body as a JSON object. The `.json()` method of your `res` variable returns a promise, which means you will need to `await` it.
+Declare a `data` variable and assign it the value of `res.json()`.
+
+# --hints--
+
+You should declare a variable named `data`.
+
+```js
+assert.match(code, /let\s*data/);
```

Comment:
same situation as earlier. since the data variable is not being reassigned we should use `const` here

```suggestion
assert.match(code, /const\s*data/);
```